docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -28,22 +30,23 @@ ALTER TABLE [<db_name>.]<tbl_name>
 alter_clause1[, alter_clause2, ...]
 ```
 
-`alter_clause` is classified into six operations: partition, rollup, schema change, rename, index, swap, comment, and compact.
+`alter_clause` is classified into operations, including rename, comment, partition, bucket, column, rollup index, bitmap index, table property, swap and compaction.
```

Comment:
-> `alter_clause` can held the following operations: rename, comment, partition, bucket, column, rollup index, bitmap index, table property, swap, and compaction.